config/vars.go

```diff
@@ -29,6 +33,8 @@ var (
 	UseReconnect = true
 	Debug        = false
 	EnableCNI    = false
+	EnableIPV4   = getEnvOrDefault("ENABLE_IPV4", "true") == "true"
+	EnableIPV6   = getEnvOrDefault("ENABLE_IPV6", "false") == "true"
```

Comment:
Maybe using command line parameters will better?

Reply:
I thought so but I'm a bit concerned about:
https://github.com/merbridge/merbridge/blob/1dcf48b8db8cffe4f6b5c616f0cc489ce1b670d7/deploy/all-in-one.yaml#L73-L79
It can't get flags from the command params, so I use env variables instead, though merbridge will clean up eBPF stuff when it exits
https://github.com/merbridge/merbridge/blob/1dcf48b8db8cffe4f6b5c616f0cc489ce1b670d7/controller/localip/localip.go#L67